.github/workflows/docs_backport_reminder.yml

```diff
@@ -0,0 +1,35 @@
+name: docs backport reminder
+
+on:
+  pull_request:
+    types: [opened, reopened, synchronize]
+    paths:
+      - 'docs/**'
+      - '.github/workflows/docs_backport_reminder.yml'
+
+jobs:
+  remind-backport:
+    name: Remind to backport docs changes
+    runs-on: ubuntu-24.04
+    permissions:
+      pull-requests: write
+    steps:
+      - name: Check if comment already exists
+        id: find-comment
+        uses: peter-evans/find-comment@b30e6a3c0ed37e7c023ccd3f1db5c6c0b0c23aad # v4.0.0
+        with:
+          issue-number: ${{ github.event.pull_request.number }}
+          body-includes: "Do the documentation changes need to be backported?"
+      - name: Add reminder comment
```

Comment:
We can add a step to check if docs/versioned_docs/.. was modified. In this case, we don't need to print the reminder.